contents/docs/how-posthog-works/queries.mdx

```diff
@@ -81,6 +81,12 @@ You may have noticed that user `3` actually upgraded from the `free` plan to the
 
 In most cases, this is exactly what we want, as it means that we can update the properties for a person without worrying about messing up our past data points. However, if instead you do want to filter based on a person's current properties, you can do so by using `person.current_properties.<property>` in HogQL or creating a [cohort](/manual/cohorts).
```

Comment:
I just saw this - and `current_person.properties.<property>` referenced in https://github.com/PostHog/posthog/pull/22812, and neither works. I think you can remove the paragraph below and just fix this example (ie. change it to `pdi.person.properties.<property>`)